aptos-move/framework/aptos-token-objects/sources/collection.move

```diff
@@ -440,6 +440,22 @@ module aptos_token_objects::collection {
         }
     }
 
+    spec increment_supply {
+        pragma aborts_if_is_partial;
+        let collection_addr = object::object_address(collection);
+        let supply = global<ConcurrentSupply>(collection_addr);
+        let post supply_post = global<ConcurrentSupply>(collection_addr);
+        aborts_if exists<ConcurrentSupply>(collection_addr) &&
+            aggregator_v2::spec_get_value(supply.current_supply) + 1 > aggregator_v2::spec_get_max_value(supply.current_supply);
```

Comment:
Can we make line breaks better here? Shouldn't be over 100 characters

Reply:
done